### ISSUE TITLE
Monitoring doesn't display all information

### ISSUE THREAD

**Issue by AndreaBruno** [evothings/phonegap-estimotebeacons#53]
Hi,
I'm trying the beacon-finder example.
In the monitoring section I see the beacon and those are the information displayed:
State:    inside
Major:   undefined
Minor:   undefined
Identifier: EstimoteSampleRegion
UUID: myUUID
(the UUID is well displayed, this is an example)

Why can't I see the Major/Minor number?
I've test the app on Android 4.4.4 (Nexus5) and iOS 7.0.3 (Ipad Mini) but this issue remains.

**Comment by divineprog** [comment]
Hi Andrea, 

I think this is because the region is defined without major/minor numbers. It is just defined as an empty object, like this: {} which default to the standard Estimote UUID, and empty major/minor values.

I have not tested myself, but I think if you for example specify major value, that will be part of the region and available with the region info. Also specifying the minor value would mean you have a regions with one beacon in it.

When monitoring regions are reported, and if there are not major/minor in the region definition, then these fields are empty.

When ranging on the other hand, you specify a region to range for, but the result is actual beacons ranged, and they have major/minor values.

You can find some more info about ranging/monitoring here: http://evothings.com/how-to-develop-beacon-apps-in-javascript-with-evothings-studio-and-estimote/

Does this help? Let me know how it goes!

Best regards, Mikael

**Comment by AndreaBruno** [comment]
Hi Mikael,

thank you so much for your answer.
Yes, you are right, I'm using the default Region with empty major/minor values.
the link you shared was very useful!

Do you know if it is possibile to change the Monitor time interval?

Best regards,
Andrea

**Comment by divineprog** [comment]
@AndreaBruno I don't think it is possible to change the monitoring time interval. When using CoreBluetooth scanning on iOS, you can set the interval in the Objective-C API, but I have not seen anything like that for monitoring and ranging. Here is the documentation page: http://estimote.github.io/iOS-SDK/Classes/ESTBeaconManager.html

**Comment by AndreaBruno** [comment]
@divineprog I think I found the solution for Android. 
according to the Estimote SDK (as describted here: https://community.estimote.com/hc/en-us/articles/201491176-How-monitoring-in-the-background-works-on-Android-)
we can use the setBackgroundScanPeriod method of the BeaconManager to set the monitoring time interval.

Obviously, we have to edit the EstimoteBeacons.java file of the plugin and reload it.

For example, i add this line in the initialize method of EstimoteBeacons.java file and it seems to work!
"mBeaconManager.setBackgroundScanPeriod(2000, 2000);"

**Comment by divineprog** [comment]
@AndreaBruno I can add a function to call setBackgroundScanPeriod from JavaScript on Android, would that be helpful?

**Comment by AndreaBruno** [comment]
@divineprog for me it would be very helpful, of course :)

Thank you!
